vllm/worker/worker.py

```diff
@@ -202,13 +202,15 @@ def save_tensorized_model(
             tensorizer_config=tensorizer_config, )
 
     @torch.inference_mode()
-    def determine_num_available_blocks(self) -> Tuple[int, int]:
+    def determine_num_available_blocks(self, other_model_memory_usage=0) -> Tuple[int, int]:
         """Profiles the peak memory usage of the model to determine how many
         KV blocks may be allocated without OOMs.
 
         The engine will first conduct a profiling of the existing memory usage.
         Then, it calculate the maximum possible number of GPU and CPU blocks
         that can be allocated with the remaining free memory.
+        
+        other_model_memory_usage is the memory usage of the other model, eg. proposal model in speculative decoding
```

Comment:
Please use the formal docstring. Also I'd suggest make this argument more general, not limiting to the draft models.

```suggestion
        Args:
            other_memory_usage: The other the memory usage in bytes, such as the draft model in speculative decoding.
```